test/JDBC/input/views/sys-all_views.sql

```diff
@@ -0,0 +1,77 @@
+CREATE DATABASE db_all_views
```

Comment:
Is it possible to incorporate these test cases in prepare/verify/cleanup format?

Reply:
Is this what you. mean?  (https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/362/files#diff-b54c68fdb9c4844875af3b42f2dec54a8e5775e92ecc843fba66c4b35bd5ec99R10-R53) These test cases are in the prepare/verify/cleanup format, but they're using different object names.

Reply:
Oh since these test cases are already in the prepare/verify/cleanup format, I believe I can just remove this file to avoid being redundant (similar to what @shalinilohia50 said here: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/399#discussion_r925361418).  I'll change that in the next commit.